velox/exec/NestedLoopJoinProbe.cpp

```diff
@@ -477,7 +477,7 @@ RowVectorPtr NestedLoopJoinProbe::genCrossProductSingleBuildVector(
     probeRowCount_ = 1;
   } else {
     probeRowCount_ = std::min(
-        (vector_size_t)outputBatchSize_ / buildRowCount,
+        static_cast<vector_size_t>(outputBatchSize_ / buildRowCount),
```

Comment:
nit: I see that this cast is redundant.

Reply:
@aditi-pandit Can you fix this nit? Thanks!

Reply:
Done. Thanks !